docs/README.md

```diff
@@ -80,6 +80,14 @@ Please refer to this [page](ingress/ingress.md) for details on how AKO supports
 
 Please refer to this [page](objects.md) for details on how AKO interprets the Kubernetes objects and translates them to Avi objects.
 
+### AKO Compatibility Guide
+AKO version 1.5.1 support for Kubernetes, Openshift, Avi Controller is as below:
+
+| **Orchaestrator/ Controller** | **Versions Supported** |
+| --------- | ----------- |
+| `Kubernetes` | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| `Openshift` | 4.3, 4.4, 4.5, 4.6, 4.7 |
+| `Avi Controller` | 20.1.3, 20.1.4, 20.1.5, 20.1.6 |
```

Comment:
I think it'll be better to specify a range of versions rather than individual ones. A range also covers patches as well.

Reply:
To make it in similar way as mentioned previously AVI docs, I have mentioned all values instead of range as doc team will pick up content from here.